adapters/rest-adapter-generator/src/main/java/uk/gov/justice/services/adapters/rest/generator/JaxRsInterfaceGenerator.java

```diff
@@ -73,8 +72,10 @@
     private TypeSpec createInterfaceFor(final Resource resource) {
         final TypeSpec.Builder interfaceSpecBuilder = interfaceSpecFor(resource);
 
-        resource.getActions().values().forEach(action ->
-                interfaceSpecBuilder.addMethods(forEach(action)));
+        resource.getActions().values().forEach(action -> {
```

Comment:
Quite right...I suppose it depends whether we want to throw an IllegalStateException and stop processing the RAML or just ignore other types.  I will go for ignoring the other types and filter.